framework/include/controls/PIDControl.h

```diff
@@ -0,0 +1,55 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+// MOOSE includes
+#include "Control.h"
+
+// Forward declarations
+class PIDControl;
+
+template <>
+InputParameters validParams<PIDControl>();
```

Comment:
I dont need this anymore do I

Reply:
Correct - for new objects we don't setup the legacy parameters. You can also remove the forward declarations above on 15-16; these were only needed with the old legacy params